webpack.config.js

```diff
@@ -13,17 +13,34 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(j|t)s$/,
-        exclude: /node_modules/,
+        parser: {
+          system: false
+        }
+      },
+      {
+        test: /\.m?(js|ts|tsx)$/,
+        exclude: /(node_modules|bower_components)/,
         use: "babel-loader"
+      },
+      {
```

Comment:
This module rule is unneeded and can be removed. It wouldn't work if we attempted to use it because we don't have style-loader or css-loader installed.

Reply:
The comment suggests removing the css loader, but the area marked and associated with the comment is unclear. I think you mean remove this:

`{
        test: /\.css$/,
        use: [
          { loader: "style-loader" },
          {
            loader: "css-loader",
            options: {
              modules: true
            }
          }
        ]
      }`

Reply:
See [webpack docs](https://webpack.js.org/configuration/module/#rule) for what I mean by "module rule".

You are right that it refers to the entire object that contains the `test`